django/http/request.py

```diff
@@ -187,9 +193,10 @@ def get_raw_uri(self):
         Return an absolute URI from variables available in this request. Skip
         allowed hosts protection, so may return insecure URI.
         """
+        _, _, host_header = self._parsed_host_header
         return '{scheme}://{host}{path}'.format(
             scheme=self.scheme,
-            host=self._get_raw_host(),
+            host=host_header,
```

Comment:
since this is now a reconstructed host header, it will be different from the provided host header e.g. in the case the host header has a default port included: `Host: example.com:443` this will now be `example.com`

Reply:
I don't think so, my reconstruction keeps the port if what was specified explictly:
See https://github.com/django/django/blob/9fbac5064df4eb5902c48ec3760c282da6d844f4/django/http/request.py#L112 for retrieving the host/port and https://github.com/django/django/blob/9fbac5064df4eb5902c48ec3760c282da6d844f4/django/http/request.py#L125 for reconstructing it. Or do I miss something?